pyproject.toml

```diff
@@ -27,7 +27,6 @@ dependencies = [
     "beautifulsoup4~=4.9.3",
     "bleach-allowlist~=1.0.3",
     "bleach~=3.3.0",
-    "cairocffi==1.2.0",
```

Comment:
IIRC this was deliberately pinned. @netchampfaris ?

It's not a dependency on WeasyPrint https://github.com/Kozea/WeasyPrint/blob/master/pyproject.toml

Reply:
It is in 52.5, the currently pinned version 😄 
https://github.com/Kozea/WeasyPrint/blob/v52.5/setup.cfg

Reply:
always a good sign to fewer system dependencies 😄

Reply:
Reverted this change. Will handle WeasyPrint upgrade in a separate PR.